edit-post/components/header/plugins/style.scss

```diff
@@ -0,0 +1,42 @@
+.components-choice-menu-plugins {
```

Comment:
It looks like `components-choice-menu-plugins` and `components-choice-menu-plugins__label` are never used.